bin/heroku-tailscale-start.sh

```diff
@@ -39,14 +44,12 @@ else
   TAILSCALE_HOSTNAME="$TAILSCALE_HOSTNAME"
 fi
 tailscaled -cleanup > /dev/null 2>&1
-(tailscaled -verbose ${TAILSCALED_VERBOSE:--1} --tun=userspace-networking --socks5-server=localhost:1055 > /dev/null 2>&1 &)  
+(tailscaled -verbose ${TAILSCALED_VERBOSE:--1} --tun=userspace-networking --socks5-server=localhost:1055 > /dev/null 2>&1 &)
 tailscale up \
-  --authkey="${TAILSCALE_AUTH_KEY}?preauthorized=true&ephemeral=true" \
```

Comment:
the query params here are causing a:
```console
backend error: invalid key: unable to validate API key
```

I believe that these attributes are handled in the API Key that is generated in the Tailscale admin UI